src/ipyaladin/__init__.py

```diff
@@ -1,9 +1,12 @@
 import importlib.metadata
 import pathlib
-from typing import ClassVar
```

Comment:
remove the change of typing module position (the logic is first block for built-in, second block for external libraries. And in the block follow the alphabetical order)